backend/src/main/scala/cromwell/backend/validation/ContinueOnReturnCodeValidation.scala

```diff
@@ -47,22 +51,27 @@ class ContinueOnReturnCodeValidation extends RuntimeAttributesValidation[Continu
         case Valid(ints) => ContinueOnReturnCodeSet(ints.toSet).validNel
         case Invalid(_) => invalidValueFailure(value)
       }
+    case value => invalidValueFailure(value)
   }
 
   override def validateExpression: PartialFunction[WomValue, Boolean] = {
     case WomBoolean(_) => true
     case WomString(value) if Try(value.toInt).isSuccess => true
     case WomString(value) if Try(value.toBoolean).isSuccess => true
+    case WomString(value) if value.equals("*") => true
     case WomInteger(_) => true
     case WomArray(WomArrayType(WomStringType), elements) =>
       elements forall { value =>
         Try(value.valueString.toInt).isSuccess
       }
     case WomArray(WomArrayType(WomIntegerType), _) => true
+    case _ => false
   }
 
-  override protected def missingValueMessage: String = s"Expecting $key" +
-    " runtime attribute to be either a Boolean, a String 'true' or 'false', or an Array[Int]"
+  override protected def missingValueMessage: String = "Expecting returnCodes" +
+    " runtime attribute to be either a String '*' or an Array[Int]." +
+    " Expecting continueOnReturnCode" +
+    " runtime attribute to be a Boolean, a String 'true' or 'false', or an Array[Int]"
```

Comment:
Since we no longer have different behavior for these two, can this say something like "Expecting returnCodes or continueOnReturnCode runtime attribute to be..."?